### PR TITLE
chore: include required Next.js stages to issue template

### PR DESCRIPTION
**Pull request by samcx**
Include an additional requirement in the [Issue Template](https://github.com/vercel/next.js/blob/canary/.github/ISSUE_TEMPLATE/1.bug_report.yml) for adding information about which <picture data-single-emoji=":nextjs:" title=":nextjs:">:nextjs:</picture> stage (e.g, `next dev`, `next build`, & `next start`) is affected in an issue.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 043553b9fb4d92459d3913ed7ca72e6b1f5e481a

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: .github/ISSUE_TEMPLATE/1.bug_report.yml [hunk 1]
```diff
@@ -103,6 +103,16 @@ body:
...
     validations:
       required: true
+  - type: dropdown
+    attributes:
+      label: Which stage(s) are affected? (Select all that apply)
+      multiple: true
+      options:
+        - 'next dev'
+        - 'next build'
+        - 'next start'
```
Comment (ijjk) [inline .github/ISSUE_TEMPLATE/1.bug_report.yml:113]: Should this also have an option for when deployed in production, maybe `Vercel` and then also `other` as well?

Comment (samcx) [reply .github/ISSUE_TEMPLATE/1.bug_report.yml:113]: @ijjk What do you think about this:

```
options:
  - 'next dev (local)'
  - 'next build (local)'
  - 'next start (local)'
  - 'Vercel (Deployed)'
  - 'Other (Deployed)'
```

Comment (ijjk) [reply .github/ISSUE_TEMPLATE/1.bug_report.yml:113]: Sounds good to me 👍

**Review by ijjk — APPROVED** [review]

**Pull request merged**